### PR TITLE
Graphing blog

### PR DESCRIPTION
**Pull request by Alek99**

### CODE REVIEW

**Comment by tgberkeley** [comment]
Maybe we could add in another simple non-interactive example

File: blog/2023-10-11-graphing-update.md [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+            self.stream = True
+        while self.stream:
+            for i in range(len(self.data)):
+                async with self:
+                    self.data[i]["uv"] = random.randint(0, 100)
+                    self.data[i]["pv"] = random.randint(100, 200)
+                    self.data[i]["amt"] = random.randint(200, 300)
+            await asyncio.sleep(3)
+```
+
```
Comment (picklelo) [inline blog/2023-10-11-graphing-update.md:75]: typo: reference

File: blog/2023-10-11-graphing-update.md [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+    \{"name": "E", "uv": 20, "pv": 140, "amt": 230\},
+    \{"name": "F", "uv": 40, "pv": 170, "amt": 250\},
+    \{"name": "G", "uv": 50, "pv": 190, "amt": 260\},
+]
+```
+
+Next we define a `StreamingState` class that will be used to store the data and update it with an event handler:
+
+```python
+class StreamingState(State):
```
Comment (picklelo) [inline blog/2023-10-11-graphing-update.md:56]: This should inherit from `rx.State`

Also we should use `inspect.getsource` to make sure the code matches the actual sourc

File: blog/2023-10-11-graphing-update.md [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+    data = data
```
Comment (picklelo) [inline blog/2023-10-11-graphing-update.md:57]: Can we add some comments for these vars. And also maybe add type annotations to encourage users to use them

File: blog/2023-10-11-graphing-update.md [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+                    self.data[i]["uv"] = random.randint(0, 100)
+                    self.data[i]["pv"] = random.randint(100, 200)
+                    self.data[i]["amt"] = random.randint(200, 300)
+            await asyncio.sleep(3)
+```
+
+The result is a live updating graph that looks like this:
+
+```python eval
+rx.vstack(
```
Comment (picklelo) [inline blog/2023-10-11-graphing-update.md:112]: Same here we should just do
```python eval
return index()
```
rather than rewriting the code

**Review by picklelo — COMMENTED** [review]
Looks good to me

**Review by picklelo — APPROVED** [review]

**Pull request merged**